dashboard/app/error.tsx

```diff
@@ -1,7 +1,7 @@
 "use client";
 
 import { Title, Text, Icon, Flex } from "@tremor/react";
-import { ExclamationCircleIcon } from "@heroicons/react/outline";
+import { ExclamationCircleIcon } from "@heroicons/react/24/outline";
```

Comment:
Can you fix the typo on the error message?

Reply:
just did